docs/source/app-dev/bindings-java/quickstart.rst

```diff
@@ -122,20 +122,40 @@ In this section, you will run the quickstart application and get introduced to t
 
    .. _quickstart-sandbox:
 
-#. To run the :doc:`sandbox </tools/sandbox>` (a lightweight local version of the ledger), run ``daml sandbox --dar .daml/dist/quickstart-0.0.1.dar``
+#. In 3 separate terminals, run:
 
-   The output should look like this:
+Terminal one: ``daml sandbox --port 6865``
 
-   .. code-block:: none
+Terminal two: Each of the following:
+
+- ``daml ledger upload-dar --host localhost --port 6865 .daml/dist/quickstart-0.0.1.dar``
+
+- ``daml script --ledger-host localhost --ledger-port 6865 --dar .daml/dist/quickstart-0.0.1.dar --script-name Main:initialize --output-file output.json``
```

Comment:
You’ve added the script here but it’s also still in the text below at line 164. You can’t run this twice. One of the two needs to go.